main.cpp

```diff
@@ -74,47 +97,104 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 { 
     ignoreUnused(rentalDuration, carType); //passing each function parameter to the ignoreUnused() function
     return {}; //if your function returns something other than void, add 'return {};' at the end of it.
-} 
+} 
+
 
 /*
  1)
- */
+ */
+
+float hotDogSales(int num_sold, float price)
+{ 
+    ignoreUnused(num_sold, price = 3.95f); 
+    return {}; 
+} 
 
 /*
  2)
- */
+ */
+
+int addTwoInts(int a, int b)
+{ 
+    ignoreUnused(a, b); 
+    return {}; 
+} 
 
 /*
  3)
- */
+ */
+
+int multTwoInts(int a, int b)
+{ 
+    ignoreUnused(a, b); 
+    return {}; 
+} 
 
 /*
  4)
- */
+ */
+
+double addTwoFloats(float a, float b)
+{ 
+    ignoreUnused(a, b); 
+    return {}; 
+} 
 
 /*
  5)
- */
+ */
+
+double multTwoFloats(float a, float b)
+{ 
+    ignoreUnused(a, b); 
+    return {}; 
+} 
 
 /*
  6)
- */
+ */
+
+char returnLargestChar(char a, char b, char c)
+{
+    ignoreUnused(a, b, c); 
+    return {}; 
+} 
+
 
 /*
  7)
- */
+ */
+
+void insertMagicHere(){}
 
 /*
  8)
- */
+ */
+
+void insertMoreMagicHere(int a, float b = 2.0f, bool c = false)
+{    
+    ignoreUnused(a, b, c);
+}
 
 /*
  9)
- */
+ */
+
+bool godKnows(char a = 'a', bool b = false, float c = 3.14f)
+{
+    ignoreUnused(a, b, c); 
+    return {}; 
+} 
 
 /*
  10)
- */
+ */
+
+ int convertFloatToInt (float my_float_val)
+ {
+    ignoreUnused(my_float_val);
+    return {};
+ }

```

Comment:
This function is not lined up with the indentation guidelines on replit
Fix the indentation
- [ ] revise